source/_integrations/systemmonitor.markdown

```diff
@@ -56,18 +56,27 @@ file.
 | load_1m                |                           |
 | load_5m                |                           |
 | load_15m               |                           |
-| network_in             | Interface, e.g., `eth0`   |
-| network_out            | Interface, e.g., `eth0`   |
-| throughput_network_in  | Interface, e.g., `eth0`   |
-| throughput_network_out | Interface, e.g., `eth0`   |
-| packets_in             | Interface, e.g., `eth0`   |
-| packets_out            | Interface, e.g., `eth0`   |
-| ipv4_address           | Interface, e.g., `eth0`   |
-| ipv6_address           | Interface, e.g., `eth0`   |
+| network_in             | Interface, e.g. `eth0`    |
+| network_out            | Interface, e.g. `eth0`    |
+| throughput_network_in  | Interface, e.g. `eth0`    |
+| throughput_network_out | Interface, e.g. `eth0`    |
+| packets_in             | Interface, e.g. `eth0`    |
+| packets_out            | Interface, e.g. `eth0`    |
+| ipv4_address           | Interface, e.g. `eth0`    |
+| ipv6_address           | Interface, e.g. `eth0`    |
```

Comment:
I agree the white spaces were a bad idea (I tried to align the table end pipes, but doesn't really make sense). However, I think the removed comma improves readability and the HA documentation guide also states that no Oxford commas should be used. This is no classic Oxford comma, but quite similar and not needed IMHO.

Reply:
I agree the white spaces might not be needed. I tried to align the table end pipes to be consistent in the editor.  Since there were already quite some white spaces between all existing entries I thought it might make sense.
I think the removed comma improves readability (a bit) and the HA documentation guide also states that no Oxford commas should be used. This is no classic Oxford comma, but quite similar and not needed IMHO.
**Update**: I read a bit more about AE and apparently there is no fixed guidance, but most people indeed put a comma after "e.g.".